data_structures/stack_using_linked_list.cpp

```diff
@@ -5,28 +5,28 @@ struct node {
     node *next;
 };
 
-node *top_var;
+node *stack_idx;
```

Comment:
this is not an index. But a pointer that is always made to point to the memory location at the top of the stack.